### PR TITLE
SetObjectMaterialText

### PR DESCRIPTION
**Pull request by AceAbhishekOfficial**
Added new warning :
Text is not updated after 16 times the functions is used , it works well in PlayerObject but in Object (Global) text is not updated.

### CODE REVIEW

File: docs/scripting/functions/SetObjectMaterialText.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Color embedding can be used for multiple colors in the text.
 :::warning
 
 You MUST use ARGB color format, not RGBA like used in client messages etc.
-
+Text does not updates after 16 times this function is used.
```
Comment (Southclaws) [inline docs/scripting/functions/SetObjectMaterialText.md:56]:
```suggestion
Text does not update after 16 calls against the same object.
```

Comment (Southclaws) [reply docs/scripting/functions/SetObjectMaterialText.md:56]: Also, is there a source for this? I've not heard of this behaviour before.

**Review by Southclaws — CHANGES_REQUESTED** [review]

File: docs/scripting/functions/SetObjectMaterialText.md (hunk 1 above)
Comment (AdamsLT) [reply docs/scripting/functions/SetObjectMaterialText.md:56]: Tested it out, can confirm - after the 16th change it stops updating.

Code used: https://pastebin.com/SmUw813m
deps: YCMD

Did it both manually, slowly and through a loop. Both stopped updating after 16. The object still gets changed (as in you can visibly see that it refreshes) but doesn't change.

Comment (AdamsLT) [reply docs/scripting/functions/SetObjectMaterialText.md:56]: Oh and can confirm another bug. Once this happens, the result of **SetObjectMaterialText** is still **1** aka OK, not **0**, as mentioned in the **Returns** section

Adjusted code to print result of the function: https://pastebin.com/xMsh5Kge

Comment (Southclaws) [reply docs/scripting/functions/SetObjectMaterialText.md:56]: Thanks for confirming that!

Comment (AceAbhishekOfficial) [reply docs/scripting/functions/SetObjectMaterialText.md:56]: Works well with PlayerObject tho

**Review by Southclaws — APPROVED** [review]

**Pull request merged**